packages/wrangler/src/user/user.tsx

```diff
@@ -1196,3 +1212,21 @@ export function getAccountFromCache():
 export function getScopes(): Scope[] | undefined {
 	return LocalState.scopes;
 }
+
+/**
+ * Make a request to the Cloudflare OAuth endpoint to get a token.
+ */
+async function fetchAuthToken(body: string) {
+	const headers: Record<string, string> = {
+		"Content-Type": "application/x-www-form-urlencoded",
+	};
+	if (getCloudflareApiEnvironmentFromEnv() === "staging") {
```

Comment:
Could you use the `domainUsesAccess` helper function here, to allow for different endpoints in the future?